### ISSUE TITLE
about CGE

### ISSUE THREAD

**Issue by xiaoxiaoxiaofufu** [zhangy0822/USER#4]
Nice work！I read your paper and codes these days, but  I only found the SGE In the embedding enhancement part, I'm curious about how CGE enhances the region-level features. Would you be kind  to share the details or provide the code? I would be very grateful.

**Comment by zhangy0822** [comment]
Thank you for your interest.
The CGE module is easy to implement, but requires you to access CLIP model from https://github.com/openai/CLIP and extract the features in advance. Specific implementation of CGE module, including nn.BatchNorm1d (512), nn.Linear (512, embed_size) nn.BatchNorm1d (embed_size), nn.Linear (embed_size, embed_size) ( Activation functions need to be added.). You can tailor it to your situation.

**Comment by xiaoxiaoxiaofufu** [comment]
I got it ,thank you !!!
